filebeat/docs/configuration.asciidoc

```diff
@@ -307,6 +307,15 @@ filebeat:
 -------------------------------------------------------------------------------------
 
 
+===== publish_async
+
+Experimental!
```

Comment:
>  By experimental, I assume we mean that customers can try the option, but their mileage may vary 

Jup. It's a trade-off between throughput and memory usage when using load-balancing. In future versions I'd consider to remove the flag and enable async mode whenever load-balancing is enabled in output plugins.